src/components/Home/HomeFooter.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Box, Typography, Link } from '@mui/material';
+
+function Copyright() {
```

Comment:
try to keep all the code format consistent! For Footer (and in most other files), you have `const Footer = {} => { ... }` to define your components. Although both serve the same purpose, keep the formatting consistent.